techniques/system/server-roles/1.0/rudder-logrotate

```diff
@@ -59,7 +59,7 @@
         create ${root_logrotate_check.syslog_file_mode} ${root_logrotate_check.syslog_user} ${root_logrotate_check.syslog_group}
         delaycompress
         postrotate
-          ${root_logrotate_check.service_restart_utility} ${root_logrotate_check.syslog_service_name} ${root_logrotate_check.syslog_rotation_method} > /dev/null || true
+          ${paths.path[systemctl]} ${root_logrotate_check.syslog_rotation_method} ${root_logrotate_check.syslog_service_name} > /dev/null || true
```

Comment:
does `systemctl rotate` exist?

Reply:
also on sles12sp1, is there a systemctl ?

Reply:
You are right, this won't work on debian based system where we are curently calling the init rotate command which send a HUP signal to syslog.
I will close this PR as it is not as straightforward as expected.

Reply:
SIGHUP is generally "systemctl reload"